ngraph_onnx/onnx_importer/utils/conv.py

```diff
@@ -37,7 +37,7 @@
     from ngraph_onnx.onnx_importer.model_wrappers import NodeWrapper
 
 
-def get_pads(onnx_node: 'NodeWrapper') -> Tuple[List[int], List[int]]:
+def get_pads(onnx_node: 'NodeWrapper', kernel_shape: 'List[int]'=None) -> Tuple[List[int], List[int]]:
```

Comment:
Why using type annotation within function signature only here?

Reply:
@postrational

Reply:
Let's stick to one convention. We use comments everywhere else, so let's move this to comments here too.
At some point we may globally switch to inline typing.